tests/components/top_bar.test.ts

```diff
@@ -298,50 +292,91 @@ describe("TopBar component", () => {
     app.destroy();
   });
 
-  test.each([
-    ["align-left", { align: "left" }],
-    ["align-center", { align: "center" }],
-    ["align-right", { align: "right" }],
-  ])("can set horizontal alignment with the toolbar", async (iconClass, expectedStyle) => {
-    const model = new Model();
-    selectCell(model, "A1");
-    const { app } = await mountParent(model);
-    const alignTool = fixture.querySelector('.o-tool[title="Horizontal align"]')!;
-    alignTool.dispatchEvent(new Event("click"));
-    await nextTick();
+  describe("horrizontal align", () => {
+    test.each([
+      ["align-left", { align: "left" }],
+      ["align-center", { align: "center" }],
+      ["align-right", { align: "right" }],
+    ])("can set horizontal alignment with the toolbar", async (iconClass, expectedStyle) => {
+      const model = new Model();
+      selectCell(model, "A1");
+      const { app } = await mountParent(model);
+      const alignTool = fixture.querySelector('.o-tool[title="Horizontal align"]')!;
+      alignTool.dispatchEvent(new Event("click"));
+      await nextTick();
 
-    const alignButtons = fixture.querySelectorAll("div.o-dropdown-item")!;
-    const button = [...alignButtons].find((element) =>
-      element.children[0]!.classList.contains(iconClass)
-    )!;
+      const alignButtons = fixture.querySelectorAll("div.o-dropdown-item")!;
+      const button = [...alignButtons].find((element) =>
+        element.children[0]!.classList.contains(iconClass)
+      )!;
 
-    button.dispatchEvent(new Event("click"));
-    await nextTick();
-    expect(model.getters.getCurrentStyle()).toEqual(expectedStyle);
-    app.destroy();
+      button.dispatchEvent(new Event("click"));
+      await nextTick();
+      expect(model.getters.getCurrentStyle()).toEqual(expectedStyle);
+      app.destroy();
+    });
+    test.each([
+      ["text", {}, "align-left"],
+      ["0", {}, "align-right"],
+      ["0", { align: "left" }, "align-left"],
+      ["0", { align: "center" }, "align-center"],
+      ["0", { align: "right" }, "align-right"],
+    ])(
+      "alignment icon in top bar matches the selected cell",
```

Comment:
`alignment icon in top bar matches the selected cell (content: %s, style: %s)`
It's useful to identify which test is failing